ngraph/core/include/ngraph/pass/pass.hpp

```diff
@@ -40,6 +40,76 @@ namespace ngraph
         typedef EnumMask<PassProperty> PassPropertyMask;
         const PassPropertyMask all_pass_property_off;
         using param_callback = std::function<bool(const std::shared_ptr<const ::ngraph::Node>)>;
+        using param_callback_map = std::map<ngraph::DiscreteTypeInfo, param_callback>;
+
+        class NGRAPH_API PassConfig
```

Comment:
Please add a documentation for this class and all public methods

Reply:
Done.